test/e2e/flowaggregator_test.go

```diff
@@ -220,6 +226,7 @@ func checkRecordsForFlows(t *testing.T, data *TestData, srcIP string, dstIP stri
 	// Iterate over recordSlices and build some results to test with expected results
 	recordSlices := getRecordsFromOutput(collectorOutput)
 	dataRecordsCount := 0
+	var TotalBytesFromSource float64
```

Comment:
we don't capitalize function-level variables

Reply:
Thanks, addressed.